code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -390,6 +442,10 @@
 	cooldown_duration = 60 SECONDS
 	///the target marked
 	var/atom/movable/marked_target
+	var/require_los = TRUE
+	var/timeout = -1
+	var/warntarget = FALSE
+	var/chargeup = -1
```

Comment:
More cursed vars. No autodoc, funny bool vars, and entirely redundant vars considering they only apply to the child type.

Reply:
autodoc added, i dont know what you mean cursed, suggestion would be appreciated i guess. And child type, its for possible later usage and makes things easier.

Reply:
Rule of thumb is you should avoid bool vars, and have them as bitflags instead. If a flag/var is only used in one place, i.e. the child proc, why are you not just adding the code unconditionally TO the child proc?